src/main/java/com/internet/shop/db/Storage.java

```diff
@@ -1,16 +1,43 @@
 package com.internet.shop.db;
 
+import com.internet.shop.model.Order;
 import com.internet.shop.model.Product;
+import com.internet.shop.model.ShoppingCart;
+import com.internet.shop.model.User;
 import java.util.ArrayList;
 import java.util.List;
 
 public class Storage {
+    public static final List<Order> orders = new ArrayList<>();
     public static final List<Product> products = new ArrayList<>();
+    public static final List<ShoppingCart> shoppingCarts = new ArrayList<>();
+    public static final List<User> users = new ArrayList<>();
+    private static Long orderId = 0L;
     private static Long productId = 0L;
+    private static Long cartId = 0L;
+    private static Long userId = 0L;
+
+    public static void addOrder(Order order) {
+        order.setId(orderId);
+        orderId++;
```

Comment:
```suggestion
        order.setId(orderId++);
```
same in other methods